src/test/java/org/gridsuite/explore/server/ExploreTest.java

```diff
@@ -81,6 +82,7 @@ public class ExploreTest {
     private Map<String, Object> specificMetadata = new HashMap<>();
     private Map<String, Object> specificMetadata2 = new HashMap<>();
     private Map<String, Object> caseSpecificMetadata = new HashMap<>();
+    private Map<String, Object> modificationSpecificMetadata = new HashMap<>();
```

Comment:
No need anymore to allocate those 4 maps at declaration point ...

Reply:
done